compiled_starters/javascript/.codecrafters/compile.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+#
+# This script is used to compile your program on CodeCrafters
+# 
+# This runs before .codecrafters/run.sh
+#
+# Learn more: <insert-link>
+
```

Comment:
If there is no compile step, we should a comment. Suggested text: 

```
# ... 
# 
# Learn more: ...

# (This file is empty since JavaScript programs don't use a compile step)
```